docs/concepts/redirects.mdx

```diff
@@ -12,133 +12,156 @@ to configure it on Ory Cloud console. If you are interested in utilizing the SDK
-below the https://sub.domain.myapp.com/only/path needs to match perfectly for
-this sub-domain and path for it to be allowed. Other redirects using myapp.com
-will fail, however, for https://anotherapp.com will succeed on any path.
+parameter on [Self-Service Flows](./self-service.mdx). For example: A user opens
+a sharable link to go to `https://myapp.com/posts`. This URL requires the user
+to be signed in and redirects the user back to the login page. To return the
+user back to the original URL append `?return_to=https://myapp.com/posts` when
+starting the self-service login flow `initSelfServiceLogin`:
+
+```js
+    "request_url": "http://<your-project>.projects.oryapis.com/self-service/login/browser?return_to=...",
+```
+
+`return_to` is set when `initSelfServiceLogin` is called in a browser with a
+`return_to` URL set. For API clients such as AJAX
+[visit this section](#api-clients).
+
+The `allow list` is used to prevent
+[Open Redirect Attacks](https://cheatsheetseries.owasp.org/cheatsheets/Unvalidated_Redirects_and_Forwards_Cheat_Sheet.html)
+by only allowing certain domains, or paths on a domain. As in the example below
+`https://sub.domain.myapp.com/only/path` needs to match the sub-domain and path
+for it to be allowed. Other redirects using `myapp.com` will fail. Redirects
+using `https://anotherapp.com` will succeed on any path.
 
 ![Browser Redirects Allow List](/images/browser-redirects/browser-redirects-allow-list.png)
 
-## Redirect Flows
+## Redirect flows
 
 Ory Cloud has a total of six flows, **Login**, **Registration**,
 **Verification**, **Recovery**, **Settings** and **Logout** which can be
 configured to redirect back to any URL. A common use case would be to redirect
-the user to your application home screen after a logout or to a specific url on
-a sub-domain after a settings password update. On project creation the redirect
-flows are set to the default Ory Cloud Managed UI. For more information on the
-Ory Managed UI, please see the [Managed User Interface](./managed-ui.mdx)
-documentation.
+the user to your application home screen after a logout or to a specific URL on
+a sub-domain after a settings password update.
 
-### Login, Registration and Settings
+On project creation the redirect flows are set to the default
+[Managed User Interface](./managed-ui.mdx).
 
-The Login and Registration flows have three fields (all optional):
-
-1. `Default Redirect URL` domain or path relative to your application URL
-2. An `Post-Password Redirect URL`
-3. An `Post-OIDC Redirect URL` (**not supported**)
+### Login, registration and settings
 
-And Settings has three fields (all optional):
+The `Login` and `Registration` flows have three fields (all optional):
 
 1. `Default Redirect URL` domain or path relative to your application URL
-2. An `Post-Password Redirect URL`
-3. An `Post-Profile Redirect URL`
+2. `Post-Password Redirect URL`
+3. `Post-OIDC Redirect URL` (_coming soon_)
```

Comment:
No "coming soon" in the docs please. Docs should describe the current state of the software.